Source/WarpX.cpp

```diff
@@ -880,13 +880,12 @@ WarpX::ReadParameters ()
                 "psatd.update_with_rho must be equal to 1 for comoving PSATD");
         }
 
-#   ifdef WARPX_DIM_RZ
-        if (!Geom(0).isPeriodic(1)) {
+        constexpr int zdir = (AMREX_SPACEDIM == 2) ? 1 : 2;
```

Comment:
A minor comment - elsewhere in the code the expression (AMREX_SPACEDIM-1) is used to get the z direction.

Reply:
Oh yes, absolutely, you're right. Will change that, have various commits to come, to fix various things. Thank you!